modules/reindex/src/test/java/org/elasticsearch/index/reindex/ReindexSourceTargetValidationTests.java

```diff
@@ -110,12 +128,16 @@ private void succeeds(RemoteInfo remoteInfo, String target, String... sources) {
     }
 
     private static IndexMetaData index(String name, String... aliases) {
+        return index(name, null, aliases);
+    }
+
+    private static IndexMetaData index(String name, @Nullable Boolean firstAliasWriteIndex, String... aliases) {
```

Comment:
nit: the name `firstAliasWriteIndex` confused me a little, since it really affects all aliases passed in the aliases array. Maybe rename to just writeIndex?

Reply:
good catch. Initially this code looked much different, and I forgot to rename it after simplifying things. Fixed in 6bd32f0